File: app/src/main/java/com/cappielloantonio/play/subsonic/models/IndexID3.java
package com.cappielloantonio.play.subsonic.models;

import com.tickaroo.tikxml.annotation.Element;
import com.tickaroo.tikxml.annotation.Xml;

import java.util.ArrayList;
import java.util.List;

@Xml
public class IndexID3 {
    @Element(name = "artist")
    protected List<ArtistID3> artists;
    protected String name;

    public List<ArtistID3> getArtists() {
        if (artists == null) {
            artists = new ArrayList<>();
        }
        return this.artists;
    }

    public void setArtists(List<ArtistID3> artists) {
        this.artists = artists;
    }

    public String getName() {
        return name;
    }

    public void setName(String value) {
        this.name = value;
    }
}

File: app/src/main/java/com/cappielloantonio/play/subsonic/models/IndexID3.kt
package com.cappielloantonio.play.subsonic.models

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml

@Xml
class IndexID3 {
    @Element(name = "artist")
    var artists: List<ArtistID3>? = null
    var name: String? = null
}